horizons/network/server.py

```diff
@@ -298,61 +300,73 @@ def onreceive(self, event):
 			return
 		except Exception as e:
-			logging.warning("[RECEIVE] Unknown or malformed packet from %s: %s!" % (player, e))
+			logging.warning("[RECEIVE] Unknown or malformed packet "
+				"from {0!s}: {1!s}".format(player, e))
 			self.fatalerror(player, __("Unknown or malformed packet. Please check your game version"))
 			return
 
 		# session id check
 		if packet.sid != player.sid:
-			logging.warning("[RECEIVE] Invalid session id for player %s (%s vs %s)!" % (peer.address, packet.sid, player.sid))
-			self.fatalerror(player, __("Invalid/Unknown session")) # this will trigger ondisconnect() for cleanup
+			logging.warning(
+				"[RECEIVE] Invalid session id for player {0!s} "
+				"({1!s} vs {2!s})!"
+				.format(peer.address, packet.sid, player.sid))
+			self.fatalerror(player, __("Invalid/Unknown session"))
+			# this will trigger ondisconnect() for cleanup
 			return
 
 		if packet.__class__ not in self.callbacks:
-			logging.warning("[RECEIVE] Unhandled network packet from %s - Ignoring!" % (peer.address))
+			logging.warning("[RECEIVE] Unhandled network packet from {0!s} - "
+				"Ignoring!".format(peer.address))
 			return
 		self.call_callbacks(packet.__class__, player, packet)
 
 
 	def onerror(self, player, packet):
 		# we shouldn't receive any errors from client
 		# so ignore them all
-		logging.debug("[ERROR] Client Message: %s" % (packet.errorstr))
+		logging.debug("[ERROR] Client Message: {0!s}".format(packet.errorstr))
 
 
 	def onfatalerror(self, player, packet):
 		# we shouldn't receive any fatala errors from client
 		# so just disconnect them
-		logging.debug("[FATAL] Client Message: %s" % (packet.errorstr))
+		logging.debug("[FATAL] Client Message: {0!s}".format(packet.errorstr))
 		self.disconnect(player.peer)
 
 
 	def onsessionprops(self, player, packet):
-		logging.debug("[PROPS] %s" % (player))
+		logging.debug("[PROPS] {0!s}".format(player))
 		if hasattr(packet, 'lang'):
 			if packet.lang in self.i18n:
 				player.gettext = self.i18n[packet.lang]
 		self.send(player.peer, packets.cmd_ok())
 
 	def oncreategame(self, player, packet):
 		if packet.maxplayers < self.capabilities['minplayers']:
-			raise network.SoftNetworkException("You can't run a game with less than %d players" % (self.capabilities['minplayers']))
+			raise network.SoftNetworkException(
+				"You can't run a game with less than {0:d} players"
+				.format(self.capabilities['minplayers']))
 		if packet.maxplayers > self.capabilities['maxplayers']:
-			raise network.SoftNetworkException("You can't run a game with more than %d players" % (self.capabilities['maxplayers']))
+			raise network.SoftNetworkException(
+				"You can't run a game with more than {0:d} players"
+				.format(self.capabilities['maxplayers']))
 		game = Game(packet, player)
-		logging.debug("[CREATE] [%s] %s created %s" % (game.uuid, player, game))
+		logging.debug("[CREATE] [{0!s}] {1!s} created {2!s}".format(
+			game.uuid, player, game))
```

Comment:
You are inconsistent at where you place the `.format(` if it doesn't completely fit the line. Sometimes its on the next line and sometimes it isnt. Please make up your mind and be consistent.